src/routes/Settings/styles.less

```diff
@@ -465,6 +465,10 @@
                         }
                     }
                 }
+
```

Comment:
@tymmesyde You can remove this as it works fine without specifying flex in the @minimum size too 👍

Reply:
468-471